.github/workflows/release-workflow.yml

```diff
@@ -201,3 +201,43 @@ jobs:
         run: |
           npm run build
           aws s3 cp build/ s3://data-product-portal-user-docs-tfhgye/ --recursive
+  deploy-on-portal-demo:
```

Comment:
Doesn't it make more sense to do this on our internal infra github? Since this is not a public demo environment?